documentation/source/newsfragments/1445.change.rst

```diff
@@ -0,0 +1 @@
+Cocotb must now be :ref:`installed <installation-via-pip>` before it can be use.
```

Comment:
```suggestion
Cocotb must now be :ref:`installed <installation-via-pip>` before it can be used.
```

Reply:
Do you know why this `ref` does not work?

Reply:
Can you point me to the rst you're trying to link to?

Reply:
I got this sorry ...